src/main/resources/com/mathworks/ci/BuildArtifactAction/summary.jelly

```diff
@@ -0,0 +1,14 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core"
+  xmlns:d="jelly:define"
+  xmlns:l="/lib/layout"
+  xmlns:t="/lib/hudson"
+  xmlns:f="/lib/form"
+  xmlns:i="jelly:fmt">
+  <t:summary icon="document.png">
+    <p><a href="buildresults">MATLAB Build Results</a></p>
+    <p>This build contains total <font color="Blue"><b>${it.totalCount} MATLAB build task(s).</b></font></p>
```

Comment:
Can we use a list instead of incomplete sentences?

This build report includes {0} tasks.
 - Number of failed tasks: {1}
 - Number of skipped tasks: {2}

@mcafaro: Can we say "build report" instead of "build"?

Reply:
Done